extra/1-radio-stations.js

```diff
@@ -12,11 +12,15 @@
  * - Create an array  starting at 87 and ending in 108
  * - Should return this array to use in other functions
  */
-
 // `getAllFrequencies` goes here
+
 function getAllFrequencies() {
-    var numbers = [87, 88, 89, 90, 91, 92, 93, 94, 95, 96, 97, 98,99,100, 101, 102, 103, 104, 105, 107,108];
-    return numbers
+    let radioArray=[];
+    for (let i=87; i<=108; i++); {
```

Comment:
Hi Donna, I just realised you have a semicolon after the rounds brackets of the for loop (line19). You will have to delete it as the semicolon is not required there.